src/no/runsafe/UserControl/database/PlayerData.java

```diff
@@ -64,21 +66,21 @@ public void setBanReason(String reason)
 		banReason = reason;
 	}
 
-	public String getBanner()
+	public UUID getBannerUUID()

```

Comment:
"banner" is kind of confusing, makes me think of actual banners - maybe use banning instead?

Reply:
that's been bugging me too, I'll refactor this